### DISCUSSION TITLE
Panel plugin not found: natel-plotly-panel

### DISCUSSION THREAD

**Discussion by shanhard** [teslamate-org/teslamate#4108] [General]
Hello, 
I am seeing an error message in Grafana under 'Charging Curve Stats' dashboard 'Panel plugin not found: natel-plotly-panel'
![Screenshot 2024-07-29 at 11 52 43](https://github.com/user-attachments/assets/99d922e5-ca75-4f37-8db1-c85abdee62f1)
I am using Teslamate v1.30.1 and Grafana is v11.0.1 (302831c125) set up on Ubuntu Server in a Docker container.

Thanks for any help to resolve this.

**Comment by shanhard** [comment]
Does anyone else have this issue?

**Comment by DrMichael** [comment]
`Charging Curve Stats` is not an 'official' dashboard. Plotly plugin was completely removed in https://github.com/teslamate-org/teslamate/pull/3382: Plotly plugin replaced with XYChart

Switch to XYChart for youre dashboard.

**Reply by shanhard** [reply]
> `Charging Curve Stats` is not an 'official' dashboard. Plotly plugin was completely removed in #3382: Plotly plugin replaced with XYChart
> 
> Switch to XYChart for youre dashboard.

Can you advise how I go about switching to XY chart? I have repulled using the cli but still have this issue.
My default 'Battery Health' dashboard is also having this error too.
![Screenshot 2024-10-18 at 16 41 27](https://github.com/user-attachments/assets/a32d8736-cd95-4734-9c83-d2fc7a691ae3)

**Comment by shanhard** [comment]
Ahh thanks for the tip... I need to update my unofficial dashboards.
